pyproject.toml

```diff
@@ -16,3 +16,8 @@ test-command = "pytest {project}/tests"
 
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "universal2", "arm64"]
+
+# Reduce the size of Linux wheels.
+[[tool.cibuildwheel.overrides]]
+select = "*-manylinux_*"
+environment = { CXXFLAGS="-g0" }
```

Comment:
Push this as a separate PR?

Reply:
Yes, I was testing stuff because this PR is very WIP-y as is 😁